ome_zarr/writer.py

```diff
@@ -198,7 +225,7 @@ def write_plate_metadata(
     group: zarr.Group,
     rows: List[str],
     columns: List[str],
-    wells: List[str],
+    wells: Union[List[str], List[dict]],
```

Comment:
This could also be `List[Union[str, dict]]` eh?

Reply:
Heh pedantic but actually correct and this reflects more the current implementation which support of list of mixed type - pushed 0ed8b0b  to unify